core/src/main/java/org/mule/runtime/core/message/ErrorTypeBuilder.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.message;
+
+import static org.mule.runtime.core.util.Preconditions.checkState;
+
+import org.mule.runtime.api.message.ErrorType;
+
+/**
+ * Builder for {@link ErrorType}. This must be the only mechanism to create an instance
+ * of {@code ErrorType}.
+ *
+ * @since 4.0
+ */
+public final class ErrorTypeBuilder {
+
+  private static final String MULE_NAMESPACE = "mule";
```

Comment:
In mule-api the runtime namespace is defined as "core" (for some javadoc). Should it be "mule" then or are they different things?